Uni_life_tracker/app/src/main/java/com/generic/plannr/Gateways/EventGateway.java

```diff
@@ -0,0 +1,152 @@
+package com.generic.plannr.Gateways;
+
+import android.annotation.SuppressLint;
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.generic.plannr.AddEventActivity;
+import com.generic.plannr.Database.DatabaseClient;
+import com.generic.plannr.Entities.Event;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EventGateway implements EventGatewayInterface {
+
```

Comment:
I think you need to merge main into your branch because now we are using event so you could implement that functionality into this branch

Reply:
done